lib/opentok.js

```diff
@@ -901,6 +901,40 @@ OpenTok = function (apiKey, apiSecret, env) {
     });
   };
 
+  /**
+   * Send DTMF digits to all participants in an active OpenTok session or to a specific client
+   * connected to that session
+   *
+   * @param sessionId The session ID corresponding to the session to which the user will connect.
+   *
+   * @param connectionId The connection ID of the stream that the DTMF signal needs to be passed to.
```

Comment:
Connection IDs are not associated with streams. (They correspond to clients connected to a session.)
```suggestion
   * @param connectionId The connection connection ID of the client you are sending the DTMF signal to.
   *  Leave this empty to send a DTMF signal to all clients connected to the session.
```

Reply:
Good catch!